data/World/Overworld.json

```diff
@@ -473,11 +584,7 @@
     {
         "region_name": "GF Outside Gate",
         "scene": "Gerudo Fortress",
-        "events": {
-            "GF Gate Open": "
```

Comment:
Curious why you removed the logical access into GF from this entrance. I assume the actor is still there to open the gate though?

Reply:
It's not relevant logically, because you can only open it as adult, but if you're adult you can just get to the other side in logic.

Reply:
In ER? I thought this would let adult in when it's the first GF access, or did that Wasteland -> Fortress entrance stop being shuffled?

Reply:
This region is already in Gerudo Fortress, it's just the other side of the gate.

Reply:
I think I'm just confused. Can't ER have an entrance become a "dead-end" by being replaced with `Wasteland Near Fortress -> GF Outside Gate`? Until the gate is opened from the other side? I can still see it being relevant logically if starting with Prelude, Ocarina, Membership card, and going from ToT -> LLR -> GF Outside Gate, for example. How's this entrance being handled now? Or how do you get to the other side in logic as adult?

Reply:
You can just get to the other side as adult by getting caught, which corresponds to the `is_adult` in the exit to "Gerudo Fortress".

Reply:
Don't you stop getting caught when you have the membership card? Ah, but that doesn't matter for logic here, I got it. Logic can always pass via `is_adult`, but in actual play, you can't get caught if you have the card but you can open the gate.